src/types.ts

```diff
@@ -47,8 +47,9 @@ export interface RevalidateOptionInterface {
   dedupe?: boolean
 }
 
-type keyFunction = () => string | any[] | null
-export type keyInterface = keyFunction | string | any[] | null
+export type keyType = string | any[] | null
```

Comment:
```suggestion
export type keyType = string | number | string[] | number[] | null
```

Reply:
I think this is okay. A cache **key** should be required to be a string, but if you pass a number it's okay and will be converted.

Reply:
I'm having bugs with `key === 0` in my previous example 🙈 maybe swr is using `!` to check?

Reply:
Yep SWR is using `!` and `0` is falsy.